Vet clinic database: Query multiple tables

MNisarAli opened the pull request:
## 🏷️ Creating, Modifying Tables, & Querying Data with JOIN

This pull request contains the following modifications to the vet_clinic database:

### Added Features ⚡:
- Created two new tables named 'owners' and 'species' with specified columns.
- Modified 'animals' table with foreign keys.
- Populated the 'owners' and 'species' tables with the provided data.
- Added data for the 'animals' table based on specified conditions.
- Wrote SQL queries (using JOIN) to answer given questions per [project requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_multiple_tables.md#project-requirements)

### Changes Made 🔄:

- Added new SQL queries to the data.sql file to insert data into 'owners' and 'species' tables.
- Modified schema.sql file to create 'owners' and 'species' tables and modify 'animals' table.
- Added new SQL queries to the queries.sql file to answer the given questions.

### Screenshots 📸:

![image](https://user-images.githubusercontent.com/101400298/224369448-4d670996-a950-44e6-af3d-3fa73f3502ef.png)
![image](https://user-images.githubusercontent.com/101400298/224369583-0702b19e-d298-410f-a95c-e2d89dfee47d.png)

MNisarAli committed:
Initialize new tables & establish relationships

MNisarAli committed:
Add owners & species, update animal owners & species

MNisarAli committed:
Add queries using join for animal-related data

pinkmoon25 reviewed (approved):
Hi @MNisarAli , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
gitflow :heavy_check_mark: 
Schema :heavy_check_mark: 
Data :heavy_check_mark: 
Queries :heavy_check_mark: 

## Optional suggestions

**[OPTIONAL] In order to keep our ReadMe file professional I suggest that you remove the Live demo section since this is a database project and currently the link in the Readme is broken and giving a 404 error** :+1: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

MNisarAli committed:
Removed Live Demo link as suggested by reviewer

MNisarAli commented:
Hi @PinkMoon25,
Thank you for taking the time to review my project and providing your valuable feedback. I have implemented the suggestion and removed the Live demo section from the ReadMe 🧾 file to maintain professionalism. Thank you 🎊 for bringing this to my attention and for your helpful feedback. Your input was 🙌 greatly appreciated and has helped me improve my work.
